evm/src/cpu/decode.rs

```diff
@@ -55,8 +56,6 @@ const OPCODES: [(u8, usize, bool, usize); 32] = [
     (0xf6, 0, true, COL_MAP.op.get_context),
     (0xf7, 0, true, COL_MAP.op.set_context),
     (0xf9, 0, true, COL_MAP.op.exit_kernel),
-    (0xfb, 0, true, COL_MAP.op.mload_general),
```

Comment:
I'd add a comment for clarity (maybe like what's been done above for logic flags)